hotsos/core/ycheck/engine/properties/requires/common.py

```diff
@@ -49,6 +50,33 @@ def __iter__(self):
             yield item
 
 
+class PackageCheckItems(CheckItemsBase):
+
+    @cached_property
+    def packages_to_check(self):
+        return [item[0] for item in self]
+
+    @cached_property
```

Comment:
this should be an abc.abstractproperty but to do this we'll need to change:

class CheckItemsBase(object):

to be 

class CheckItemsBase(abc.ABC):

Reply:
can we also rename info() to packaging_helper()